.flake8

```diff
@@ -1,2 +1,3 @@
 [flake8]
 ignore = E203, W503
```

Comment:
Should we have this match stratus? It has a much longer list.

Reply:
I can see why it would be nice to have them match but seeing as this is an open source project, I'd rather keep it as minimal as possible, adding things we run into as we go along